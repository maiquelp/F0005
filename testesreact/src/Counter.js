import React from 'react';

function Counter(count) {
  return (
    <div className="Counter">
        <p>
          <h1>Edit <code>{count * 2}</code> and save to reload.</h1>
        </p>
    </div>
  );
}

export default Counter;
