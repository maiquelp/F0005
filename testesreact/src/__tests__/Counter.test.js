import React from 'react';
import { shallow } from 'enzyme';
import Counter from '../Counter';

describe('Testing Counter', () => {
  it('should render properly', () => {
    const wrapper = shallow(
      <Counter count={5} />
    );
    expect(wrapper).toMatchSnapshot();
    wrapper.setProps({count: 20});
    expect(wrapper).toMatchSnapshot();
  });
});