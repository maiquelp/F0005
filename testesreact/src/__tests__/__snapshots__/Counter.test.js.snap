// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Counter should render properly 1`] = `ShallowWrapper {}`;

exports[`Testing Counter should render properly 2`] = `ShallowWrapper {}`;
